src/compass/schemas/s1_cslc_geo.yaml

```diff
@@ -93,6 +95,12 @@ range_split_spectrum_options:
   # Bandwidth of the high-band sub-image
   high_band_bandwidth: num(min=0, required=False)
 
+lut_options:
+  # LUT spacing in range direction in meters
+  range_spacing: num(min=1, required=False)
+  # LUT spacing in azimuth direction in meters
+  azimuth_spacing: num(min=1, required=False)
```

Comment:
Not familiar with the schema, but `min=1` means the minimum value(s), or minumum **number** of the value? if the former, I would suggest to decrease it further in accordance with the default value in `s1_cslc_geo.yaml` (i.e. 0.028 seconds)

Reply:
Goof catch. I will decrease it to zero so that we can plug in only positive numbers